game/game.go

```diff
@@ -0,0 +1,28 @@
+package game
+
+// Game contains the game tree information
+type Game struct {
+	Root *Node
+}
+
+// NewGame creates a Game struct
+func NewGame() *Game {
```

Comment:
Nit: because the package is Game you can just say New(), because it'll be called as `game.New()`